lib/routes/situations.js

```diff
@@ -39,4 +39,7 @@ module.exports = function(api) {
 
     api.route('/situations/:situationId/openfisca-request').get(allowShow, situations.openfiscaRequest);
 
+    api.route('/situations/:situationId/openfisca-test').get(situations.openfiscaTest);
+    api.route('/situations/:situationId/openfisca-test').post(situations.openfiscaTest);
```

Comment:
Oh yes. That's a very good idea to rely on `api.route('/situations/:situationId/openfisca-request')`, to manage _decorated OpenFisca request_ and allow different formats depending on the request `Content-Type`.

However, in the current context, the structure of _OpenFisca requests_ and of _OpenFisca test files_ are too different to be served by the same endpoint without creating too much confusion (for the developer in the code and for the endpoint consumer).

Let's serve on a different endpoint for now. Once OpenFisca refactorings (Web API, YAML test format) have occurred, we can reconsider this option.